src/app.postcss

```diff
@@ -332,3 +332,8 @@ input.required:invalid,
 .label.required input:invalid {
   @apply text-error-500-400-token;
 }
+
+.link {
```

Comment:
the `anchor` class exists for this. I don't think it bolds though.

Reply:
Sweet, I'll switch to anchor and underline then.